### PR TITLE
Update actions, documentation templates and configuration files to v0.1.1

### PR DESCRIPTION
**Pull request by juanpedromoreno**
> ❕This is an automated PR created from a release in [`47degrees/.github`](https://github.com/47degrees/.github).

# What has been done in this PR?

Update Github Actions workflows, documentation templates and configuration files to v0.1.1.

# Included changes

- Notes of the previous version can be found [here](https://github.com/47degrees/.github/releases/tag/v0.1.0)
- Fix cp command

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,10 +1,69 @@
...
+!.vscode/tasks.json
+!.vscode/launch.json
+!.vscode/extensions.json
+
+### sbt-fix ###
+# Scalafix & Scalafmt configuration files are automatically downloaded by `sbt-fix`
+# https://github.com/47degrees/sbt-fix
+
+.scalafix.conf
+.scalafmt.conf
```
Comment (BenFradet) [inline .gitignore]: should we remove it?

Comment (franciscodr) [reply .gitignore]: It seems these files are managed by sbt-fix so we wouldn't need to keep them updated in the repo.

Comment (juanpedromoreno) [inline .gitignore]:
```suggestion
```

Comment (juanpedromoreno) [reply .gitignore]: Fixed in 4845551

**Pull request merged**